src/fedora/27/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM fedora:27
+
+# Workaround to avoid NuGet restore timeout
+RUN dnf upgrade -y nss \
+    && dnf clean all
+
+# Install the base toolchain we need to build anything (clang, cmake, make and the like)
+# this does not include libraries that we need to compile different projects, we'd like
+# them in a different layer.
+RUN dnf install -y \
+        clang \
+        cmake \
+        lldb-devel \
+        llvm-devel \
+        make \
+        python \
+        which \
+    && dnf clean all
+
+# Install tools used by the VSO build automation.
+RUN dnf install -y \
+        git \
+        npm \
+        nodejs \
+        tar \
+        zip \
+    && dnf clean all
+
+RUN npm install -g azure-cli --unsafe-perm
+
+# Dependencies of CoreCLR and CoreFX.
+RUN dnf install -y \
+        glibc-locale-source \
+        libcurl-devel \
+        libgdiplus \
+        libicu-devel \
+        libunwind-devel \
+        libuuid-devel \
+        lttng-ust-devel \
+        openssl-devel \
+        uuid-devel \
+    && dnf clean all
+
+RUN dnf install -y compat-openssl10-devel --allowerasing
+
+# Until Fedora.27-x64 official packages are available, we have to restore the ubuntu ones instead.
+ENV __PUBLISH_RID=ubuntu.14.04-x64
```

Comment:
NIT: I thought the syntax for 'ENV' required a space instead of an '=' but if this has the desired effect and compiles then that should be fine.

Reply:
It can go either way, with or without the equals sign.

Reply:
https://docs.docker.com/engine/reference/builder/#env - both are supported, when `=` is used you can set more than one environment variable with a single `ENV` instruction.

```
ENV myName="John Doe" myDog=Rex\ The\ Dog \
        myCat=fluffy
```